packages/circuit-ui/components/NotificationToast/NotificationToast.tsx

```diff
@@ -79,30 +79,36 @@ const iconMap = {
   notify: Notify,
 };
 
-type NotificationToastWrapperProps = HTMLAttributes<HTMLDivElement> & {
+type NotificationToastWrapperProps = {
   variant: Variant;
 };
 
 const toastWrapperStyles = ({
   theme,
   variant,
 }: NotificationToastWrapperProps & StyleProps) => css`
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-  position: relative;
   background-color: ${theme.colors.bodyBg};
-  padding: ${theme.spacings.kilo} ${theme.spacings.mega};
   border-radius: ${theme.borderRadius.byte};
   border: ${theme.borderWidth.mega} solid ${theme.colors[colorMap[variant]]};
   overflow: hidden;
-  transition: opacity 200ms ease-in-out, height 200ms ease-in-out,
-    visibility 200ms ease-in-out;
+  will-change: height;
```

Comment:
Did you notice a change when adding this? Curious

Reply:
I think this would only be noticeable on older hardware.